dev-packages/cli/src/theia.ts

```diff
@@ -144,7 +144,16 @@ function rebuildCommand(command: string, target: ApplicationProps.Target): yargs
         })
         .command({
             command: 'download:plugins',
-            handler: () => downloadPlugins()
+            describe: 'Download defined external plugins.',
+            builder: {
+                'preserveVsix': {
```

Comment:
maybe better `unpack` which is false by default?

Reply:
I think we discussed that perhaps we wanted the preservation of the `.vsix` file as to be the one which requires the flag to be turned on (requires an application to actually request such a feature), since most clients would it expect it to unpack?

The only client I know as of now that'd want to preserve is the `theia-electron` AppImage.
cc @marcdumais-work

Reply:
Ah, that’s fine I just wanted to have shorter option name ) but it is also not so important

Reply:
I agree :) I updated the flag so it is shorter.
I use `-packed` or `-p` to control the behavior when downloading plugins.

Reply:
> The only client I know as of now that'd want to preserve is the `theia-electron` AppImage.

Not only AppImage but potentially all OS-specific packagings for Electron to be safe (.deb, .rpm, .msi, .exe, .dmg, ...). 

TL;DR: we noticed that the [Java Language Support for Visual Studio Code by Red Hat](https://github.com/redhat-developer/vscode-java) extension writes to the extension's directory at runtime. This fails if the extension is unpacked in such a way that the Electron app user does not have write access to it, which happens when installing a packaged Electron app (except maybe if packages as simple .tar or .zip and extracted by end-user). 

We are not sure if other VS Code extensions may also be also writing to their directory at runtime, so to be safe I would keep Electron-bundled extensions packed in `.vsix` format. 

(note: I did not see this behavior on writing in one's own folder at runtime for built-in extensions)